src/components/list-address/AddressList.jsx

```diff
@@ -0,0 +1,39 @@
+import "./index.scss"
+
+const AddressList = ({ addresses, addAddress, removeAddress}) => {
+    const handleAddress = () => {
+        const address = prompt('Ingrese una nueva direccion ')
```

Comment:
Please, DO NOT use prompt, that's a security flaw and it's not recommended.

Reply:
Rodrigos's component